ansible_3par_docker_plugin/tasks/create_conf_file.yml

```diff
@@ -54,7 +54,7 @@
     no_log: True
 
   - name: get the iscsi ports (vlan tagged)
-    local_action: shell /usr/bin/sshpass -p {{ INVENTORY[item]['hpe3par_password'] }} ssh -oStrictHostKeyChecking=no {{ INVENTORY[item]['hpe3par_username'] }}@{{ INVENTORY[item]['hpe3par_ip'] }} "showport -iscsivlans" | awk '{print $3}' | sed -n '1!p'
```

Comment:
In the getports -iscsivlans response, if the VLAN is not set (set to '-'), we ignore that as they are unresponsive (ping fails to connect)